modules/externalipam/externalipam_plugin.rb

```diff
@@ -0,0 +1,11 @@
+require 'externalipam/phpipam/phpipam_plugin'
+require 'externalipam/netbox/netbox_plugin'
+
+module Proxy::Ipam
+  class Plugin < ::Proxy::Plugin
+    plugin :externalipam, ::Proxy::VERSION
+    uses_provider
+    default_settings use_provider: 'externalipam_phpipam'
```

Comment:
Default setting should be probably nil - error out during startup, the proxy stack will automatically disable the plugin. We have a validation framework for that, just look on how other modules are implemented.